examples/basic-express-typescript/route.ts

```diff
@@ -0,0 +1,27 @@
+
+import * as express from 'express'
+import * as parsers from 'body-parser'
+import * as cors from 'cors'
+import * as sync from '../../fh-sync'
+
+const router = express.Router()
```

Comment:
No semis? Daring.

Reply:
I'm _trying_ it [out here](https://github.com/evanshortiss/env-var/blob/master/package.json#L12) too 😂